src/lap_stats/lap_stats.c

```diff
@@ -1,24 +1,46 @@
-#include "lap_stats.h"
+/*
+ * Race Capture Pro Firmware
+ *
+ * Copyright (C) 2016 Autosport Labs
+ *
+ * This file is part of the Race Capture Pro fimrware suite
+ *
+ * This is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details. You should
+ * have received a copy of the GNU General Public License along with
+ * this code. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include "auto_track.h"
 #include "dateTime.h"
-#include "gps.h"
-#include "geopoint.h"
 #include "geoCircle.h"
 #include "geoTrigger.h"
-#include "loggerHardware.h"
+#include "geopoint.h"
+#include "gps.h"
+#include "lap_stats.h"
+#include "launch_control.h"
 #include "loggerConfig.h"
-#include "modp_numtoa.h"
-#include "modp_atonum.h"
+#include "loggerHardware.h"
 #include "mod_string.h"
+#include "modp_atonum.h"
+#include "modp_numtoa.h"
 #include "predictive_timer_2.h"
 #include "printk.h"
 #include "tracks.h"
-#include "test.h"
 
 #include <stdint.h>
 
-#include "auto_track.h"
-#include "launch_control.h"
-
+/* Make the radius 3x the size of start/finish radius.*/
+#define GEO_TRIGGER_RADIUS_MULTIPLIER 3
```

Comment:
it's weird, I don't see it elsewhere in the PR, yet it seems to be first added here.